README.md

```diff
@@ -47,28 +47,38 @@ git submodule update --remote --merge
 yarn build
 ```
 
-### Deploy bridge
+### Deploy
 
-The bridge deploy script expects two environment variables. 1) address of the bridge controller: `BRIDGE_CONTROLLER_ADDRESS` and 2) address of the Toucan contract registry: `TOUCAN_CONTRACT_REGISTRY_ADDRESS`
+Copy `.env.example` to `.env` and set the desired configuration in `.env`.
 
 #### Testnets/Matic
 
 Before you deploy you should set up your signer wallet. You can either pass a mnemonic via the environment variable `MNEMONIC` or create a `mnemonic.txt` file with the seed phrase in the root of the repository.
 
+```shell
+yarn hardhat run scripts/deploy.js --network [matic|mumbai]
```

Comment:
Why don't we have the deployment code under `deploy/`?  I thought that was the expected place for it.

Reply:
I think `scripts` is as expected or even more than `deploy` given that Hardhat docs use `scripts` for their examples[1]. Btw `deploy` is limiting as it excludes upgrade, or any other non-deploy related scripts. Given how we have structured `tokenizer`, `deploy` to me feels like a place to put deployment-related logic (not necessarily top-level) and `scripts` holds top-level scripts (that may or may not use code from `deploy`, given that as I understand it, `deploy` is standardized by `hardhat-deploy`).

[1] https://hardhat.org/hardhat-runner/docs/guides/deploying

Reply:
https://github.com/wighawag/hardhat-deploy#the-deploy-task says:

![image](https://user-images.githubusercontent.com/100738/215365095-9da6b387-896f-4489-b893-7c005478ec98.png)

so we could configure it to be something else, but I don't see the benefit of doing that.  The intended way of selecting a subset of the deployment scripts to run is by using [`--tags`](https://github.com/wighawag/hardhat-deploy#deploy-scripts-tags-and-dependencies).  I'm not sure I follow your point about why putting deploy scripts in `deploy/` is limiting?  Upgrades could go elsewhere, and certainly non-deploy related scripts must not go in `deploy/`.